### PR TITLE
fix part size value appropriately in upload_snowball_objects() API

### PR DESCRIPTION
**Pull request by erwin-vanduijnhoven**
This pull request fixes the issue mentioned in #1332

### CODE REVIEW

File: minio/api.py [hunk 1]
```diff
@@ -2769,15 +2769,17 @@ def upload_snowball_objects(self, bucket_name, object_list, metadata=None,
         else:
             length = os.stat(name).st_size
 
+        part_size = max(length, 5 * (1024) ** 2)
```
Comment (balamurugana) [inline minio/api.py:2772]: This fix is `part_size = 0 if length < MIN_PART_SIZE else length`

**Review by balamurugana — CHANGES_REQUESTED** [review]

File: minio/api.py (hunk 1 above)
Comment (erwin-vanduijnhoven) [reply minio/api.py]: Updated the PR accordingly.

**Review by balamurugana — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Comment by kkzzhhqq** [comment]
Hello, my minio version is 7.2.0 and I installed it using pip install. The prompt indicates that it is already the latest version, but the current version still has this issue

**Comment by balamurugana** [comment]
> Hello, my minio version is 7.2.0 and I installed it using pip install. The prompt indicates that it is already the latest version, but the current version still has this issue

This PR is not part of the release

**Comment by kkzzhhqq** [comment]
> > Hello, my minio version is 7.2.0 and I installed it using pip install. The prompt indicates that it is already the latest version, but the current version still has this issue
> 
> This PR is not part of the release

@balamurugana Will this PR be released in the official version?

**Comment by balamurugana** [comment]
> > > Hello, my minio version is 7.2.0 and I installed it using pip install. The prompt indicates that it is already the latest version, but the current version still has this issue
> > 
> > 
> > This PR is not part of the release
> 
> @balamurugana Will this PR be released in the official version?

Stay tuned!

**Comment by kkzzhhqq** [comment]
> > > > Hello, my minio version is 7.2.0 and I installed it using pip install. The prompt indicates that it is already the latest version, but the current version still has this issue
> > > 
> > > 
> > > This PR is not part of the release
> > 
> > 
> > @balamurugana Will this PR be released in the official version?
> 
> Stay tuned!

Okay, could you please let me know when it is expected to be released?

**Comment by harshavardhana** [comment]
> Okay, could you please let me know when it is expected to be released?

There is no SLA @kkzzhhqq for community issues - we may release it when we have the extra bandwidth

If you are looking for expedited releases you should check our subscription for your production requirements https://min.io/pricing

**Pull request merged**